recipes/r-packrat/build.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# R refuses to build packages that mark themselves as Priority: Recommended
+mv DESCRIPTION DESCRIPTION.old
+grep -v '^Priority: ' DESCRIPTION.old > DESCRIPTION
```

Comment:
Can you please remove above three lines in all build scripts? They are only needed for core packages (and those are all packaged now).